src/Density.jl

```diff
@@ -31,3 +32,62 @@ function check_density_real(ρreal)
         @warn "Large imag(ρ)" norm_imag=norm(imag(ρreal))
     end
 end
+
+"""
+Interpolate a function expressed in a basis `b_in` to a basis `b_out`
+This interpolation uses a very basic real-space algorithm, and makes a DWIM-y attempt to take into account the fact that b_out can be a supercell of b_in
+TODO move this outside of density
+"""
+function interpolate_density(ρ_in::Density, b_in::PlaneWaveBasis, b_out::PlaneWaveBasis)
```

Comment:
`Density` already contains a basis object, can't you just use that?
Also: I feel `interpolate_density(b_in, b_out, ρ_in) more natural, however, `interpolate_density(ρ_in, b_out)` ...

Reply:
Absolutely, I missed this, thanks!